code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -786,7 +762,7 @@ generate/load female uniform sprites matching all previously decided variables
 					break
 
 // Only renders the head of the human
-/mob/living/carbon/human/proc/update_body_parts_head_only()
+/mob/living/carbon/human/proc/update_body_parts_head_only(var/update_limb_data)
```

Comment:
```suggestion
/mob/living/carbon/human/proc/update_body_parts_head_only(update_limb_data)
```

Reply:
We removed the lint for a reason. Ike likes having var/ in proc args.